src/code/client.coffee

```diff
@@ -334,11 +334,15 @@ class CloudFileManagerClient
       @saveFileDialog stringContent, callback
 
   saveFile: (stringContent, metadata, callback = null) ->
-    if metadata?.provider?.can 'save', metadata
+    if metadata?.provider?.can('save') and not metadata.forceSaveDialog
```

Comment:
I didn't see that you added the extra parameter in the lara provider and I thought it was a copy-paste error.  I'll add it back.